api/auth/jwtUtils.js

```diff
@@ -1,49 +1,24 @@
-const jwt = require('jsonwebtoken');
 const logger = require('../logger')('jwtUtils');
-
-const { SESSION_LIFETIME_MINUTES, SESSION_SECRET } = process.env;
-
-const TOKEN_ISSUER = 'CMS eAPD API';
-const HS256 = 'HS256';
-
-/**
- * Returns a signed JWT with content as the payload. Inverse function for
- *   verifyWebToken(token).
- * @name signWebToken
- * @param {Object} payload - payload of information to be stored within the JWT
- * @returns {String} signed JWT
- */
-const signWebTokenOptions = {
-  algorithm: HS256,
-  expiresIn: `${SESSION_LIFETIME_MINUTES || 5}m`,
```

Comment:
Two of the features the existing auth flow had was (1) the ability to invalidate a current user's session from the back-end, as well as (2) timeout a session. I'm wondering if we still have those abilities. If not, we should address them in this PR or via additional GitHub issues.

Reply:
I'm not sure if we have that ability in Okta. I would guess we probably do, but that's not something we were actually using from the front end so I would say we address that with the admin panel in a new ticket.